src/app/series/directives/series-podcast.component.html

```diff
@@ -96,6 +96,12 @@
       </publish-fancy-field>
     </publish-fancy-field>
 
+    <publish-fancy-field label="Enclosure Prefix Url" textinput [model]="podcast" name="enclosurePrefix">
+      <div class="fancy-hint">
+        The enclosure prefix URL for your podcast feed. This URL may be used to set a redirect prefix on audio requests.
+      </div>
```

Comment:
Does this need further info (perhaps without specifically endorsing or specific to podtrac or blubrry?)